api/src/Service/ReportStatusService.php

```diff
@@ -170,6 +170,10 @@ public function getMoneyInState()
     {
         if ($this->report->hasMoneyIn()) {
             return ['state' => self::STATE_DONE, 'nOfRecords' => count($this->report->getMoneyTransactionsIn())];
+        } elseif ($this->report->getMoneyInExists() && $this->report->getReasonForNoMoneyIn()) {
```

Comment:
It looks like you might have missed the `!` before `$this->report->getMoneyInExists()`.

I am assuming that we use the state done if they haven't put any money in but have provided a reason for no money in.

A similar issue for the last elseif statement where we mark it as incomplete if they haven't put any money in.

Reply:
Once the new pages are wired up, there will also be an answer for the 'getMoneyExists' property - it would be a 'yes' or 'no'.

- The first statement accounts for a deputy reporting money in items.
- The second statement accounts for when a deputy has an answer for the getMoneyExists (which would be 'no') question and getReasonForNoMoneyIn - the state would be marked as done.
- The final statement accounts for when a deputy answers the 'getMoneyExists' question but hasn't entered a reason yet - the state would be marked as incomplete

Reply:
Agreed to rename the 'getMoneyInExists' property to avoid confusion